addon/components/au-time-picker.js

```diff
@@ -1,16 +1,35 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
-import { tracked } from '@glimmer/tracking';
 import { isPresent } from '@ember/utils';
+import { localCopy } from 'tracked-toolbox';
+import { formatTimeDigit } from '../helpers/format-time-digit';
 
 export default class AuTimePickerComponent extends Component {
-  @tracked hourValue = this.args.hours || 12;
-  @tracked minuteValue = this.args.minutes || 0;
-  @tracked secondValue =
-    isPresent(this.args.showSeconds) && !this.args.showSeconds
-      ? 0
-      : this.args.seconds || 0; // ignore 'seconds' argument if 'showSeconds' is false
-  @tracked keyCodes = [8, 9, 13, 33, 34, 37, 39, 46];
+  @localCopy('args.hours') hourValue;
+  @localCopy('args.minutes') minuteValue;
+  @localCopy('args.seconds') secondValue;
+
+  constructor() {
+    super(...arguments);
+    this.hourValue = this.hourValue || 12;
```

Comment:
localCopy also supports initializers so you can set a default value that way I think? https://github.com/tracked-tools/tracked-toolbox#localcopy

Reply:
Also, I think we want to add some validation here since at the moment it's possible to pass in values outside of the allowed range. Passing in `30` to the `@hours` arg would display that value in the input field that way.

Maybe we can add asserts here in case arg values aren't strings, or when they are outside the range? That will probably catch most cases. It wouldn't detect invalid input when the arg values are updated but that might be fine for now?